pkg/platform/kube/test/suite.go

```diff
@@ -0,0 +1,182 @@
+/*
+Copyright 2017 The Nuclio Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/nuclio/nuclio/pkg/cmdrunner"
+	"github.com/nuclio/nuclio/pkg/common"
+	"github.com/nuclio/nuclio/pkg/functionconfig"
+	"github.com/nuclio/nuclio/pkg/platform"
+	nuclioioclient "github.com/nuclio/nuclio/pkg/platform/kube/client/clientset/versioned"
+	"github.com/nuclio/nuclio/pkg/platform/kube/controller"
+	"github.com/nuclio/nuclio/pkg/platform/kube/functionres"
+	"github.com/nuclio/nuclio/pkg/platformconfig"
+	processorsuite "github.com/nuclio/nuclio/pkg/processor/test/suite"
+
+	"github.com/ghodss/yaml"
+	"k8s.io/client-go/kubernetes"
+)
+
+type KubeTestSuite struct {
+	processorsuite.TestSuite
+	CmdRunner   cmdrunner.CmdRunner
+	RegistryURL string
+	Controller  *controller.Controller
+}
+
+func (suite *KubeTestSuite) SetupSuite() {
+	if !common.GetEnvOrDefaultBool("NUCLIO_K8S_TESTS_ENABLED", false) {
+		suite.T().Skip("Test can only run when `NUCLIO_K8S_TESTS_ENABLED` environ is enabled")
```

Comment:
"environ" -> "env var" or "environment variable" or just, nothing at all (clear from context this is env var)
:)